From 2dc48561e4f69692c052d9edcb514f322ca02ece Mon Sep 17 00:00:00 2001
From: yasibena <118657398+yasibena@users.noreply.github.com>
Date: Fri, 13 Dec 2024 12:21:13 +0330
Subject: [PATCH 01/28] add gitignore

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..0a95508
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,2 @@
+.idea/
+.vscode/
\ No newline at end of file
-- 
2.43.0.windows.1

