From 950dd0aada0f6f29c857ec399691e8f2338bf743 Mon Sep 17 00:00:00 2001
From: yasibena <118657398+yasibena@users.noreply.github.com>
Date: Sun, 15 Dec 2024 21:00:48 +0330
Subject: [PATCH 06/28] add dynamic route example

---
 src/app/page.tsx | 34 ++++++++++++++++++++++++++++++++--
 1 file changed, 32 insertions(+), 2 deletions(-)

diff --git a/src/app/page.tsx b/src/app/page.tsx
index f02bb06..720cf94 100644
--- a/src/app/page.tsx
+++ b/src/app/page.tsx
@@ -1,5 +1,35 @@
-export default function Home() {
+type Post = {
+  id: number;
+  title: string;
+};
+
+export default async function Home() {
+  const res: Response = await fetch("https://jsonplaceholder.org/posts", {
+    cache: "no-store",
+  });
+
+  const posts: Post[] = await res.json();
+
   return (
-   <h1>!سلام دنیا</h1>
+    <main>
+      <h1>سلام رفیق!</h1>
+      <p>
+        لورم ایپسوم متن ساختگی با تولید سادگی نامفهوم از صنعت چاپ، و با استفاده
+        از طراحان گرافیک است، چاپگرها و متون بلکه روزنامه و مجله در ستون و
+        سطرآنچنان که لازم است، و برای شرایط فعلی تکنولوژی مورد نیاز، و کاربردهای
+        متنوع با هدف بهبود ابزارهای کاربردی می باشد، کتابهای زیادی در شصت و سه
+        درصد گذشته حال و آینده، شناخت فراوان جامعه و متخصصان را می طلبد، تا با
+        نرم افزارها شناخت بیشتری را برای طراحان رایانه ای علی الخصوص طراحان
+        خلاقی، و فرهنگ پیشرو در زبان فارسی ایجاد کرد، در این صورت می توان امید
+        داشت که تمام و دشواری موجود در ارائه راهکارها، و شرایط سخت تایپ به پایان
+        رسد و زمان مورد نیاز شامل حروفچینی دستاوردهای اصلی و جوابگوی سوالات
+        پیوسته اهل دنیای موجود طراحی اساسا مورد استفاده قرار گیرد.
+      </p>
+      <ul>
+        {posts.map((post: Post) => (
+          <li key={post.id}>{post.title}</li>
+        ))}
+      </ul>
+    </main>
   );
 }
-- 
2.43.0.windows.1

